code formatting changes

jfrerich opened the pull request:
profile pic is now 40x40

update section header

paddingHorizontal is now 20

swap username and displayname and adjust font sizes for each

set the user row height
set the displayname height

use justifyContent: center instead of hardcoding vertical padding<!-- Thank you for contributing a pull request! Here are a few tips to help you:

1. If this is your first contribution, make sure you've read the Contribution Checklist https://developers.mattermost.com/contribute/getting-started/contribution-checklist/
2. Read our blog post about "Submitting Great PRs" https://developers.mattermost.com/blog/2019-01-24-submitting-great-prs
3. Take a look at other repository specific documentation at https://developers.mattermost.com/contribute
-->

#### Summary
<!--
A brief description of what this pull request does.
-->

#### Ticket Link
<!--
If this pull request addresses a Help Wanted ticket or fixes a reported issue, please link the relevant GitHub issue, e.g.

  Fixes https://github.com/mattermost/mattermost-mobile/issues/XXXXX

Otherwise, link the JIRA ticket.
-->

#### Checklist
<!--
Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.
-->
- [ ] Added or updated unit tests (required for all new features)
- [ ] Has UI changes
- [ ] Includes text changes and localization file updates
- [ ] Have tested against the 5 core themes to ensure consistency between them.

#### Device Information
This PR was tested on: <!-- Device name(s), OS version(s) -->

#### Screenshots
<!--
If the PR includes UI changes, include screenshots/GIFs/Videos (for both iOS and Android if possible).
-->

#### Release Note
<!--
Add a release note for each of the following conditions:

* New features and improvements, including behavioural changes, UI changes
* Bug fixes and fixes of previous known issues
* Deprecation warnings, breaking changes, or compatibility notes

If no release notes are required write NONE. Use past-tense. Newlines are stripped.

Example:

```release-note
Added a new config setting ServiceSettings.FooBar. Added a new column Foo to the Users table.
```

```release-note
NONE
```
-->

```release-note

```

matthewbirtch commented on app/components/user_list_row/index.tsx:
```diff
@@ -55,12 +56,13 @@ const getStyleFromTheme = makeStyleSheetFromTheme((theme) => {
-        displayName: {
-            fontSize: 15,
+        username: {
+            fontSize: 12,
```
We should use the typography helper here. It's not using right font right now

`...typography('Body', 75, 'regular'),`

matthewbirtch commented on app/components/user_list_row/index.tsx:
```diff
@@ -55,12 +56,13 @@ const getStyleFromTheme = makeStyleSheetFromTheme((theme) => {
-        username: {
-            fontSize: 15,
+        displayName: {
+            fontSize: 16,
```
We should use the typography helper here. It's not using right font right now

`...typography('Body', 200, 'regular'),`

matthewbirtch reviewed (changes requested):
Thanks @jfrerich. A added a few notes on a few lines of code, but one other note about the checked state:

1. The selector icon on the right side of the row looks like it needs a few updates. 
- For the unchecked state, it should be using the compass icon `circle-outline` with `color: changeOpacity(theme.centerChannelColor, 0.32),` rather than using a border. 
- For the checked state, it should be using the compass icon `check-circle` and it should be using the `theme.buttonBg` color. 

2. When `Deactived` displays it is showing on a 3rd row of text which looks a bit awkward to me. I wonder if we can just add it in brackets on the same line as the username instead. e.g. `@matthew.birtch (Deactivated), Alternatively it could be good to show it the same way we show `GUEST` or `BOT` in a tag. That might make more sense. Thoughts? I also question why a deactivated user would show up at all in the New DM modal though. We should look at that.
<img width="381" alt="image" src="https://user-images.githubusercontent.com/2040554/197416908-b23823ad-863c-440a-8ef7-6cc83b26470f.png">

3. When I scroll down the user list, the loader isn't always visible while it is loading a new page of users. Can we get it so that there is enough space at the bottom to show a loader while it's loading the next page?

4. When I search, as soon as I start typing for a split second it shows the empty state that says 'No matches found'. Can we prevent this from flashing until we are certain there are no results? It feels confusing to show me this the second I start typing. We should just show the loader until we are sure there are no results. Not sure if this is an issue with this PR or an issue with all search inputs like this.
<img width="400" src="https://user-images.githubusercontent.com/2040554/197417254-180fb718-27c5-4111-9de5-4e5c215ad501.gif">

There are other elements, but it sounds like we'll work on the chips and the bottom panel for when users are selected in other PRs, so I'll wait for follow-ups to comment on those things.

jfrerich commented:
My guess is that haveing having a `Guest` and `Deactivated` tag on the same line would be too much? We could argue that it doesn't matter that a deactivated user was a guest, but that is up for debate.

jfrerich commented:
>There are other elements, but it sounds like we'll work on the chips and the bottom panel for when users are selected in other PRs, so I'll wait for follow-ups to comment on those things.

This is correct. I have broken the changes in to smaller ticket in hopes of reducing the amount of PR growth experienced in the search screen :)

jfrerich commented:
@matthewbirtch, I fixed your issue 1).

For 2, 3, and 4, I suggest we create tickets for now.

I also fixed a but when a user has selected the max number of users. Now the opacity of the radio button changes to opacity of 0.16. This was in code but not getting used.

matthewbirtch commented:
> @matthewbirtch, I fixed your issue 1).
> 
> For 2, 3, and 4, I suggest we create tickets for now.

Sure, that's fine by me.

> I also fixed a but when a user has selected the max number of users. Now the opacity of the radio button changes to opacity of 0.16. This was in code but not getting used.

@jfrerich for this state, I think we should grey out the entire row - just like in the [design](https://www.figma.com/file/uCS4dYCHSS5NO7PjlVUBzI/MM-19947-Mobile-Multi-select?node-id=1836%3A19026)
<img src="https://user-images.githubusercontent.com/2040554/197529775-9d813d58-da20-4b3b-a36e-8dc38cf4cb11.png" width="414">

jfrerich commented:
@matthewbirtch, I made the change to add opacitcy of 0.32 to the entire row if it is disabled.

larkox reviewed (approved):
LGTM

avinashlng1080 commented on app/components/user_list_row/index.tsx:
```diff
@@ -107,6 +91,9 @@ const getStyleFromTheme = makeStyleSheetFromTheme((theme) => {
     };
 });
 
+const disabledOpacity = 0.32;
+const defaultIconOpacity = 0.32;
```
good to keep the standard for constants
